File: Data Structures & Algorithms/max-consecutive-ones/submission-0.cs
public class Solution {
    public int FindMaxConsecutiveOnes(int[] nums) {
        int length = nums.Length;
        int count = 0;
        int maxCount = 0;
        for(int i = 0 ; i < length ; i++)
        {
            if(nums[i] == 1)
            {
                count+=1;
            }
            else
            {
                count = 0;
            }
            if(count > maxCount) maxCount = count;
        }
        return maxCount;
    }
}

File: Data Structures & Algorithms/single-number/submission-0.js
class Solution {
    /**
     * @param {number[]} nums
     * @return {number}
     */
    singleNumber(nums) {
        let map = new Map()
        for(let num of nums){
            map.set(num,(map.get(num) || 0) + 1)
        }
        for(let [num,count] of map){
            if (count === 1){
                return num
            }
        }
    }
}
